File: TEst/TEst/ViewController.m
//
//  ViewController.m
//  TEst
//
//  Created by 陈欢 on 2018/3/21.
//  Copyright © 2018年 陈欢. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    UIButton *btn = [UIButton buttonWithType:UIButtonTypeSystem];
    btn.frame = CGRectMake(0, 0, 100, 100);
    [btn setTitle:@"ooooooooooo" forState:UIControlStateHighlighted];
    //UIColor *btn =   [btn titleColorForState:UIControlStateNormal];
    //[btn setAttributedTitle:@"iiii" forState:UIControlStateNormal];
    [btn setTitleColor:[UIColor redColor] forState:UIControlStateHighlighted];
    btn.reversesTitleShadowWhenHighlighted = YES;
    btn.titleEdgeInsets = UIEdgeInsetsMake(0, 0, 0, 0);
    [self.view addSubview:btn];
    
    UIImage *image = [[UIImage alloc]init];
    UIImage *hhh = [UIImage imageWithContentsOfFile:@"oooo"];
    // Do any additional setup after loading the view, typically from a nib.
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: sevent/sevent/ViewController.swift
//
//  ViewController.swift
//  sevent
//
//  Created by 陈欢 on 2017/11/18.
//  Copyright © 2017年 陈欢. All rights reserved.
//

import UIKit

class ViewController: UIViewController,UIScrollViewDelegate {

    var imageView:UIImageView!
    
    var scrollView:UIScrollView!
    
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        imageView = UIImageView(frame: self.view.bounds)
        imageView.image = UIImage(named: "samplePhoto.jpeg")
        imageView.isUserInteractionEnabled = true
        scrollView = UIScrollView(frame: self.view.frame)
        scrollView.maximumZoomScale = 4.0
        scrollView.minimumZoomScale = 1
        scrollView.backgroundColor = UIColor.black
        scrollView.contentSize = imageView.bounds.size
        scrollView.delegate = self;
        self.view.addSubview(scrollView)
        scrollView.addSubview(imageView);
        
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
    func viewForZooming(in scrollView: UIScrollView) -> UIView? {
        return imageView
    }


}

